README.md

```diff
@@ -1,2 +1,18 @@
 # xpdSchema
 Schema for XPD metadatastore documents
+
+##Notes
+An important thing to note is the "server/client" key convention.
+Servers contain the data which the key describes. 
+For example data which has the key `detector_calibration_server_uid`
+contain the images/other data needed to calibrate the detector.
+Data which has the `detector_calibration_client_uid` rely on the 
```

Comment:
Subsequent datasets that will use this calibration are `clients` that hold a reference to the `server` with the correct calibration data by having the same value for `detector_calibration_client_uid` as the `detector_calibration_server_uid` of the `server`.